app/controllers/ems_cloud_controller.rb

```diff
@@ -160,7 +160,7 @@ def ems_cloud_form_fields
                      :provider_id                     => @ems.provider_id ? @ems.provider_id : "",
                      :hostname                        => @ems.hostname,
                      :api_port                        => @ems.port,
-                     :api_version                     => @ems.api_version,
+                     :api_version                     => @ems.api_version ? @ems.api_version : "v2",
```

Comment:
@Fryguy right, but for new provider, it's just EmsCloud, cause you don't know the type yet. So first thing you do is pick the type in selectbox, but it doesn't shoot request back to this code. I think it was like this in the non angular version?

So should I define the default value on EmsCloud class?